features/polyExplorer/src/components/dataSharingGauge/dataSharingGauge.css

```diff
@@ -90,7 +90,7 @@
 }
 
 .data-sharing-gauge-average-label {
-    position: absolute;
+    position: relative;
```

Comment:
I am not sure how the exact position has been calculated, but I think this should be back to `position: absolute`, since it positions the label exactly where the "Average" line is.

Reply:
Yes, thanks! Already recognized and changed it, but still have to push it